jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/editor/S3TreeCellRenderer.kt

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.services.s3.editor
+
+import com.intellij.icons.AllIcons
+import com.intellij.ui.ColoredTreeCellRenderer
+import com.intellij.ui.LoadingNode
+import com.intellij.ui.SimpleTextAttributes
+import com.intellij.ui.speedSearch.SpeedSearchUtil
+import javax.swing.JComponent
+import javax.swing.JTree
+import javax.swing.tree.DefaultMutableTreeNode
+
+class S3TreeCellRenderer(private val speedSearchTarget: JComponent) : ColoredTreeCellRenderer() {
+    init {
+        myUsedCustomSpeedSearchHighlighting = true
```

Comment:
this is hideous - but whatever works.

Reply:
Yaaaa.... It uses the wrong jcomponent normally so it never finds anything to highlight. As to the name, I just work here :P